pkg/cmd/server/serve.go

```diff
@@ -239,7 +251,7 @@ func ensureDefaultProfile(server *server.Server, apiPort uint32) error {
 	})
 }
 
-func startLocalRunner(params bootstrap.LocalRunnerParams) error {
+func EnsureRunnerRegistered() error {
```

Comment:
No need for these methods to be public. Leftover from the previous idea, I think.